include/oneapi/dpl/pstl/hetero/dpcpp/sycl_traits.h

```diff
@@ -94,10 +94,10 @@ struct __is_heap_check;
 template <typename _Predicate, typename _ValueType>
 struct __create_mask_unique_copy;
 
-template <class _Tag, typename _ExecutionPolicy, typename _Tp, typename>
+template <class _Tag, typename _Tp, typename>
 struct __brick_fill;
 
-template <class _Tag, typename _ExecutionPolicy, typename _Tp, typename>
+template <class _Tag, typename _Tp, typename>
```

Comment:
To remove the last `, typename`.

Reply:
Fixed

Reply:
Issue created: https://github.com/uxlfoundation/oneDPL/issues/2180